app/src/main/res/layout/fragment_select_region.xml

```diff
@@ -1,77 +1,84 @@
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/cw_big_logo" />
 
-    <Spinner
-        android:id="@+id/spinner_regions"
-        style="@style/Widget.AppCompat.Spinner.Underlined"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginVertical="@dimen/padding_normal"
-        android:backgroundTint="@color/white"
-        android:paddingVertical="@dimen/padding_normal"
-        android:textSize="24sp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/select_regions_label" />
+        <TextView
+            android:id="@+id/select_regions_label"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/padding_large"
+            android:layout_marginBottom="@dimen/padding_normal"
+            android:text="@string/select_region_label"
+            android:textAlignment="center"
+            android:textColor="@color/white"
+            android:textSize="19sp"
+            android:textStyle="bold"
+            app:layout_constraintBottom_toTopOf="@id/spinner_regions"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/region_logo"
+            app:layout_constraintVertical_bias="0.1" />
 
-    <TextView
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        android:layout_marginTop="@dimen/padding_normal"
-        android:ellipsize="end"
-        android:lineSpacingExtra="3sp"
-        android:text="@string/privacy_select_region"
-        android:textColor="@color/white"
-        android:textSize="14sp"
-        app:layout_constraintBottom_toTopOf="@id/btn_continue"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/spinner_regions"
-        app:layout_constraintVertical_bias="0" />
+        <Spinner
+            android:id="@+id/spinner_regions"
+            style="@style/Widget.AppCompat.Spinner.Underlined"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginVertical="@dimen/padding_normal"
+            android:backgroundTint="@color/white"
+            android:paddingVertical="@dimen/padding_normal"
+            android:textSize="24sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/select_regions_label" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
```

Comment:
Just wrapping with `NestedScrollView` layout doesn't work completely correct with this screen unfortunately.
We need to use `wrap_content` here to fix the problem.

Could you please verify the layouts by using the smallest layout previews in Android Studio and increasing the text size?

Reply:
layout_height changed to wrap_content

tested on device (Pixel 2 XL) with sysytem font set to Largest, system display size set to Largest, and app put into split screen